backend/src/main/java/com/alex/eshop/service/CategoryService.java

```diff
@@ -6,13 +6,11 @@
 import com.alex.eshop.exception.DataNotFoundException;
 import com.alex.eshop.mapper.CategoryMapper;
 import com.alex.eshop.repository.CategoryRepository;
+import jakarta.transaction.Transactional;
```

Comment:
replace `import jakarta.transaction.Transactional;`
with `import org.springframework.transaction.annotation.Transactional;`
you have this import in several classes